src/components/Navbar/NavBar.tsx

```diff
@@ -298,8 +339,66 @@ const ReactNavLink = styled(Link)`
   :hover {
     color: #9e9e9e;
   }
+  position: relative;
 `;
 
 const Close = styled(CloseIcon)`
   cursor: pointer;
-`;
\ No newline at end of file
+`;
+
+const Dropdown = styled.div`
+  z-index: 11;
+  left: 141px;
+  top: 19px;
+  width: 213px;
+  height: 183px;
+  a:hover {
+    color: #9e9e9e;
+  }
+  background-color: #ffffff;
+  position: absolute;
+`;
+
+const DropdownMobile = styled.div`
+  width: 375px;
+  height: 163px;
+  position: relative;
+  bottom: 8px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+`;
+const DropItem = styled.div`
+  margin-top: 18px;
+  margin-left: 28px;
+  font-size: 13px;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.05em;
+  text-decoration: none;
+  color: black;
+`;
+const DropdownItem = DropItem.withComponent('a');
+const DropdownItemLink = DropItem.withComponent(Link);
+const MobileItem = styled(DropdownItem)`
+  margin: 0px;
+  color: #949494;
+`;
+const MobileItemLink = styled(DropdownItemLink)`
+  margin: 0px;
+  color: #949494;
+`;
+const DropdownButtonContainer = styled.div`
+  display:flex;
+  position:relative
```

Comment:
Can you run yarn eslint --fix to fix these whitespace issues? There should be a space after the :